packages/contentful/migrations/gp2/outputs/20230531101009-date-fields.js

```diff
@@ -0,0 +1,37 @@
+module.exports.description = 'Updates enums';
+
+module.exports.up = (migration) => {
+  const outputs = migration.editContentType('outputs');
+
+  outputs.changeFieldControl('addedDate', 'builtin', 'datePicker', {
+    ampm: '12',
+    format: 'time',
+    helpText:
+      'Date output was shared with GP2 Network (different from publication date)',
+  });
+  outputs.changeFieldControl('lastUpdatedPartial', 'builtin', 'datePicker', {
+    ampm: '12',
+    format: 'time',
+    helpText: 'Does not include changes to Publish Date and Admin notes',
+  });
+};
+
+module.exports.down = (migration) => {
+  const outputs = migration.editContentType('outputs');
```

Comment:
I'm confused with this down function 🤔
Why are we changing the validations on documentType and type since we're just updating the field control for dates on the up function?

Reply:
good point. I've removed them